packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>0.0.0.20250124</version>
+    <version>0.0.0.20250204</version>
```

Comment:
I have just merged a PR that updated this package, so there is now a merge conflict here. @thack1 can you please fix it?

Reply:
fixed